README.md

```diff
@@ -84,6 +84,15 @@ npm publish
 npm push origin && npm push origin v[N.N.N]
 ```
 
+## Interop release process for when breaking changes are introduced
+
+1. Get branches of go-ipfs and js-ipfs working together in interop locally using environment variables to point at the local versions
+2. In this repo make a branch containing the interop changes as well as setting the go/js-ipfs commit hashes to be used by CI. Merge to master.
+3. Update go-ipfs and js-ipfs branches to use interop/master instead of whatever is released
+4. Release go and js-ipfs (not necessarily together) and after each is released (or RC'd) bump interop to use the release instead of the commit hash
+5. Release interop
+6. Bump go and js-ipfs to use released interop instead of master
```

Comment:
If we want to simplify and have go-ipfs and js-ipfs green, instead of this repo, then maybe:

```suggestion
1. Get branches of go-ipfs and js-ipfs working together in interop locally using environment variables to point at the local versions
2. In this repo make a branch containing the interop changes. CI won't pass with release versions of go/js-ipfs, but it is ok, as long (1) passed locally. Merge to master.
3. Update go-ipfs and js-ipfs branches to use a commit from interop/master instead of whatever is released. The ipfs/interop test in their CI should be green now.
4. Release go and js-ipfs (not necessarily together) and after each is released (or RC'd) bump interop to use the release instead of the commit hash. This will fix CI in ipfs/interop repo.
5. Release ipfs/interop  when its CI is green again.
6. Bump go and js-ipfs to use released interop instead of master
```

Reply:
Assuming no disastrous problems with the js monorepo I think making CI pass seems like the reasonable thing to do here and if it's as simple as changing a line from `v0.X.Y` -> `abcd1234` then it's pretty simple as well.

In the go-ipfs v0.11.0 release alone there are 3 breaking changes that it's helpful to see fail and then fix individually (sharding, circuit-relay, pubsub). If we don't keep CI green here it puts a lot of pressure on reviewers to make sure when things work on their machines that it's not just on their machines due to some particular configuration or stray environment variable.

Reply:
Ack. After thinking about it in context of go-ipfs 0.11 using `interop/master` in go-ipfs (rolling branch, instead of hardcoded revision) will make our merge fest easier.